### ISSUE TITLE
Dark theme broken again

### ISSUE THREAD

**Issue by RockMum** [dtinth/dark-facebook#70]
This one has been problematic since Social Fixer started, which is  a shame. I know it will most likely be fine later today, or pretty soon but it is frustrating that only this theme is so buggy, but all the other themes are so unattractive. I think SF and Thai need a powwow to sort this out once and for all.

**Comment by dtinth** [comment]
Have you tried the alternative URL and/or alternative methods described here?

https://github.com/dtinth/dark-facebook/wiki/Troubleshooting:-Theme-Stops-Working-Completely-on-Social-Fixer

**Comment by RockMum** [comment]
Yep, done that, thanks.  It works really well that.  Social Fixer doesn't, mainly because it takes him ages to fix anything. Your theme is always broken. You might want to have a word with him, but don't expect him to be very helpful.  Head up his own arse that one.
